extensions/wikia/WikiFactory/Close/SpecialCloseWiki_body.php

```diff
@@ -202,9 +202,7 @@ private function doProcess() {
 		$valid = true;
 		$newWiki = 0;
 		if ( isset($this->mRedirect) && !empty($this->mRedirect) ) {
-			if ('http://' == substr($this->mRedirect, 0, 7)) {
-				$this->mRedirect = str_replace('http://', '', $this->mRedirect);
-			}
+			$this->mRedirect = preg_replace( '!^https?://!', '', $this->mRedirect );
 			Wikia::log( __METHOD__, "check domain {$this->mRedirect}" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace or remove this deprecated Wikia::log logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)

[project-id]: wikia:app